src/components/NewFeatures/NewFeatures.js

```diff
@@ -63,6 +63,12 @@ const NewFeatures = () => {
             </Link>
           </div>
         </div>
+        <div className="dp-block-wlp dp-box-shadow m-t-36">
+          <div style={{ marginLeft: '20px' }}>
```

Comment:
maybe I could use this className = "m-l-18" o "m-l-24" and thus not use inline style. the atomic classes of the library jump from 6px to 6px

Reply:
👍